### PR TITLE
Add perineurial cell

### LINKED ISSUE #1726: [NTR]perineurial cell
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label**
perineurial cell

**Synonyms** (add reference(s), please)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A cell that belongs to the supporting tissue surrounding a nerve fiber bundle. It has thin long bipolar cytoplasmic processes, pinocytotic vesicles, fragments of external lamina and/or external lamina-like material, attachment plaques, and desmosome-like junctions. [ NCI ] 

NCIT:C41442 http://purl.obolibrary.org/obo/NCIT_C41442

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl)
perineural cell http://purl.obolibrary.org/obo/CL_0002576
based on placement in NCIT perineural cell http://purl.obolibrary.org/obo/NCIT_C41416

**What is the anatomical structure that the cell is a part of?** (check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon)
perineurium 

**Your ORCID**
0000-0001-7655-4833

**Additional notes or concerns**
HuBMAP Skin ASCT+B table term

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #1726

### CODE REVIEW

**Review by dosumis — CHANGES_REQUESTED** [review]
Looks good. Maybe worth adding a ref, e.g. https://link.springer.com/article/10.1134/S002209302201001X

**Comment by dosumis** [comment]
Also - might also be same as perineurial/peripheral glial cell:

image
https://link.springer.com/article/10.1134/S002209302201001X

Maybe worth adding as comment for now if not clear enough that can be added as synonym or sufficient for reflecting in classification.

**Comment by shawntanzk** [comment]
> Maybe worth adding as comment for now if not clear enough that can be added as synonym or sufficient for reflecting in classification.

I will do this, i think peripheral glial is too vague esp when there are like schwann cells and all

**Comment by shawntanzk** [comment]
> 


> image
> 
> https://link.springer.com/article/10.1134/S002209302201001X
> Maybe worth adding as comment for now if not clear enough that can be added as synonym or sufficient for reflecting in classification.

Oh the above is perineural which is different from perineurial

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28558,13 +28558,13 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) r
...
 EquivalentClasses(obo:CL_4023187 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0013615)))
 
-# Class: obo:CL_4023211 (perineurial cells)
+# Class: obo:CL_4023211 (perineurial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NCIT:C41442"^^xsd:string) obo:IAO_0000115 obo:CL_4023211 "A myofibroblast cell that belongs to the supporting tissue surrounding a nerve fiber bundle. It has thin long bipolar cytoplasmic processes, pinocytotic vesicles, fragments of external lamina and/or external lamina-like material, attachment plaques, and desmosome-like junctions.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NCIT:C41442"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:25818566"^^xsd:string) obo:IAO_0000115 obo:CL_4023211 "A myofibroblast cell that belongs to the supporting tissue surrounding a nerve fiber bundle. It has thin long bipolar cytoplasmic processes, pinocytotic vesicles, fragments of external lamina and/or external lamina-like material, attachment plaques, and desmosome-like junctions.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4023211 "https://orcid.org/0000-0001-7655-4833")
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:28564]: Using dce:contributor and the orcid is a string :)

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28564]: whoops soz, need to fix my protege settings for that >.<

Comment (anitacaron) [reply src/ontology/cl-edit.owl:28564]: only if it's not an orcid for dc:terms

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28564]: ah we didn't implement the "allowed" annotation properties in CL then

Comment (anitacaron) [reply src/ontology/cl-edit.owl:28564]: We did. I'll investigate it.

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See comments in linked issue.

**Comment by bvarner-ebi** [comment]
Superseded by #1832.

**Pull request closed without merging**